lib/full-site-editing/template-parts.php

```diff
@@ -60,6 +60,29 @@ function gutenberg_register_template_part_post_type() {
 }
 add_action( 'init', 'gutenberg_register_template_part_post_type' );
 
+/**
+ * Get the edit link for template parts.
+ *
+ * @param string $link    The original link.
+ * @param int    $post_id The custom post id.
+ */
+function gutenberg_get_edit_template_part_link( $link, $post_id ) {
+	$template_parts = gutenberg_get_block_templates(
+		array( 'wp_id' => $post_id ),
+		'wp_template_part'
+	);
```

Comment:
Hi, @kevin940726

This query will run for every post type. Let's return early if it's not `wp_template_part` CPT.

Do we need to run a DB query to get template part ID? Maybe we could use `_build_block_template_result_from_post` here?

P.S. I think we can also combine these two filters.

Reply:
Good idea! Updated in 9626c89.

Reply:
Thank you! Do you mind if I push some minor tweaks to align with some WP core PHP practices?

Reply:
Absolutely not! Go for it!

Reply:
Done ✅